packages/components/src/components/route-with-subroutes/route-with-subroutes.tsx

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable react/prop-types */
```

Comment:
```suggestion
```
@likhith-deriv What is this for? 🤔

Reply:
@likhith-deriv Let's keep it then for now, I will check and If I could find anything will let you know 🙇🏻🙏🏻